frontend/src/views/camp/Dashboard.vue

```diff
@@ -58,7 +64,7 @@
               :key="dayUri"
               :aria-labelledby="dayUri + 'th'"
             >
-              <tr class="day-header__row">
+              <tr :ref="days[dayUri].id" class="day-header__row">
```

Comment:
I think this should be set on the tbody and preferably as an id. Which we could reflect in the url.

Then the tbody should have a `scroll-margin-top` with the same value as the top of the `.day-header`: So the navbar does not cover the content:

![Bildschirmfoto 2024-10-14 um 08 38 49](https://github.com/user-attachments/assets/7dfacf3a-dde8-4ec1-bccc-9f6fa51c15c9)

If solved as an id, someone could then share this link and it will automatically go to the same day.